crates/miden-lib/asm/agglayer/account_components/bridge_out.masm

```diff
@@ -131,12 +131,13 @@ end
 #! - Storing the updated MMR root in account storage
 #! - Creating a BURN note with the bridged out asset
 #!
-#! Inputs:  [ASSET, INPUTS[1], INPUTS[0]]
+#! Inputs:  [ASSET, dest_network, dest_address(5)]
```

Comment:
Also the inline comments later in this procedure still say `INPUTS[1], INPUTS[0]`